From c5356c3707d45e3da5159dd955a706e1131d3bfc Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Tue, 2 Sep 2014 15:18:20 -0700
Subject: [PATCH 1/1] arch/x86/kernel: remove pr_info call when CPU tries to
 play dead

The pr_info() call when CPU tries to play dead causes system lock-up
during suspend, during the time that kernel tries to disable
non-boot CPUs.

Further investigation is needed to undestand the cause. However,
removing the pr_info() call seems to fix the lockup issue.
Remove the line now so that suspend is no longer blocked.

Reported-by: Tushar Dave <tushar.n.dave@intel.com>
Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 arch/x86/kernel/smpboot.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/x86/kernel/smpboot.c b/arch/x86/kernel/smpboot.c
index 82b2710..f0a89b9 100644
--- a/arch/x86/kernel/smpboot.c
+++ b/arch/x86/kernel/smpboot.c
@@ -1411,7 +1411,6 @@ static inline void mwait_play_dead(void)
 	 * content is immaterial as it is not actually modified in any way.
 	 */
 	mwait_ptr = &current_thread_info()->flags;
-	pr_info("cpu[%d] mwait hint: 0x%x\n", smp_processor_id(), eax);
 	wbinvd();
 
 	while (1) {
-- 
1.8.3.2

